bundle/Resources/config/services/event_listener.yml

```diff
@@ -1,6 +1,6 @@
 services:
-    netgen.ezplatform_site.event_listener.content_tagger:
-        class: Netgen\Bundle\EzPlatformSiteApiBundle\EventListener\ContentTaggerListener
+    netgen.ezplatform_site.event_listener.render_view:
```

Comment:
I feel the name of the event listener should not change here (or rather change only to `ViewTagger`). I mean, there could be more than one RenderView event in the future.

Reply:
Good point, but in that case I should also change naming of `Events::RENDER_VIEW` to reflect the meaning.

Reply:
Are you sure? Event name is one thing, you render a view, hence `Events::RENDER_VIEW`. An event *listener* is the other and it is named based on the purpose in the listener itself.

Reply:
I was thinking that `ViewRenderer`, where the event is dispatched now, is pretty specific, another render view event might happen in another context, for example if we ever need to override view manager from kernel, where the view is also rendered.

Reply:
Why should the location of the dispatched event influence the name of the event itself? Why not use the same event at both places? As a consumer of the event, I wouldn't really care where the view render comes from, what I would care only about the fact that a view is rendered.

Reply:
In the view manager example, you would not need to tag the view, since there is already another mechanism that takes care of that, so it's basically a different event. There is a dichotomy here, since you dispatch not by one but by two parameters, event name and event object. Since you subscribe to the event name, that is the actual event, and the event object is just it's payload.

Reply:
Hm, okay. Makes sense. In that case, I would deal with it when and *if* we come to it :)

Reply:
I agree, since I don't really have any good naming ideas here :)

Renamed to `ViewTaggerSubscriber` in https://github.com/netgen/ezplatform-site-api/pull/146/commits/ba17c400b002a62ccc450b916855113e79127297